erpnext/stock/doctype/item/item.js

```diff
@@ -165,11 +165,11 @@ frappe.ui.form.on("Item", {
 		frm.set_value('has_batch_no', 0);
 		frm.toggle_enable(['has_serial_no', 'serial_no_series'], !frm.doc.is_fixed_asset);
 
-		frm.call({
-			method: "set_asset_naming_series",
-			doc: frm.doc,
-			callback: function() {
+		frappe.call({
+			method: "erpnext.stock.doctype.item.item.get_asset_naming_series",
+			callback: function(r) {
 				frm.set_value("is_stock_item", frm.doc.is_fixed_asset ? 0 : 1);
+				frm.asset_naming_series = r.message;
```

Comment:
Maybe there's a better way to pass this variable to `set_asset_naming_series()`?

Reply:
Lemme check this one